basicLibrary/src/main/java/basicLibrary/Library.java

```diff
@@ -55,40 +49,39 @@ public static String hashyMappy(){
                 monthTemps.add(weeklyMonthTemperatures[i][j]);
                  if( weeklyMonthTemperatures[i][j]>high){
                      high=weeklyMonthTemperatures[i][j];
-                 }else if (weeklyMonthTemperatures[i][j]<low){
+                 }
+                 if (weeklyMonthTemperatures[i][j]<low){
                      low=weeklyMonthTemperatures[i][j];
                  }
             }
         }
-        String tempNotSeen="";
+        StringBuilder tempNotSeen= new StringBuilder(String.format("high: %d\nLow: %d", low,high));
```

Comment:
Your test is failing on this because you need to capitalize the 'H' in "high", and swap variables at the end to:  ..... , high, low));